rust/osm2lanes/src/transform.rs

```diff
@@ -349,22 +367,89 @@ pub fn get_lane_specs_ltr_with_warnings(
 
     // For living streets in Krakow, there aren't separate footways. People can walk in the street.
     // For now, model that by putting shoulders.
-    if cfg.inferred_sidewalks || tags.is(HIGHWAY, "living_street") {
+    if cfg.inferred_sidewalks || tags.highway_is("living_street") {
         if need_fwd_shoulder {
-            fwd_side.push(fwd(LaneType::Shoulder));
+            forward_side.push(LaneSpec::forward(LaneType::Shoulder));
         }
         if need_back_shoulder {
-            back_side.push(back(LaneType::Shoulder));
+            backward_side.push(LaneSpec::backward(LaneType::Shoulder));
         }
     }
+}
+
+/// From an OpenStreetMap way's tags, determine the lanes along the road from left to right.
+pub fn get_lane_specs_ltr_with_warnings(tags: &Tags, cfg: &Config) -> LaneSpecResult {
+    if let Some(spec) = non_motorized(tags, cfg) {
+        return spec;
+    }
+
+    let fwd = |lane_type: LaneType| LaneSpec {
```

Comment:
These could be replaced by `LaneSpec::forward` now. I'll make a followup change after this

Reply:
oh yeah, I forgot to finish by removing those :)